.whitesource

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Hey, why is this here?

Reply:
This code was PRed by mistake, I hadn't: refactored, considered different techniques, checked for best practices, etc.